OrbitCore/LinuxPerfData.h

```diff
@@ -17,13 +17,24 @@ class LinuxPerfData
 {
 public:
     std::string m_header = "";
-    uint32_t m_tid = 0;
+    uint32_t m_tid = 0; // TODO: This is actually redundant
     uint64_t m_time = 0;
     uint64_t m_numCallstacks = 0;
 
     CallStack m_CS;
 
     void Clear();
 
+    ORBIT_SERIALIZABLE;
+};
+
+//-----------------------------------------------------------------------------
+struct HashedLinuxPerfData
```

Comment:
HashedLinuxPerfData is a bit generic.  "HashedLinuxCallstackEvent" maybe?  And LinuxCallstackEvent instead of LinuxPerfData?  Maybe HashedLinuxPerfData could be replaced by the existing CallstackEvent struct?

Reply:
Sure, I was not aware of CallstackEvent. I changed the code, to use this struct instead. Also I renamed the name of LinuxPerfEvent to LinuxCallstackEvent